.circleci/config.yml

```diff
@@ -69,18 +73,25 @@ download_docker: &download_docker
 calculate_docker_image: &calculate_docker_image
   name: Calculate docker image
   command: |
-    git clone --quiet https://github.com/pytorch/pytorch.git "/tmp/pytorch"
     # Just a quick smoke test to see if we can actually extract the tag
+    git clone --quiet https://github.com/pytorch/pytorch.git "/tmp/pytorch"
     git -C /tmp/pytorch rev-parse HEAD:.circleci/docker
     DOCKER_TAG=$(git -C /tmp/pytorch rev-parse HEAD:.circleci/docker)
-    echo "declare -x DOCKER_IMAGE=308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-bionic-cuda10.2-cudnn7-py3.7-clang9:${DOCKER_TAG}" >> "${BASH_ENV}"
+    echo "declare -x ECR_DOCKER_IMAGE=308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/xla_base:${DOCKER_TAG}" >> "${BASH_ENV}"
```

Comment:
I am a bit confuse about the `ECR_DOCKER_IMAGE ` and `GCR_DOCKER_IMAGE `. Maybe it is worth it to add some comment somewhere in this file?